[pkg/stanza/fileconsumer] Fix long line parsing (#32100)
open-telemetry/opentelemetry-collector-contrib: receiver/filelogreceiver/README.md

| `exclude_older_than`                |                                      | Exclude files whose modification time is older than the specified [age](#time-parameters).                                                                                                                                                                      |
| `start_at`                          | `end`                                | At startup, where to start reading logs from the file. Options are `beginning` or `end`.                                                                                                                                                                        |
| `multiline`                         |                                      | A `multiline` configuration block. See [below](#multiline-configuration) for more details.                                                                                                                                                                      |

> | `force_flush_period`                | `500ms`                              | [Time](#time-parameters) since last token was emitted from the file, after which a partial log at the end of the file may be emitted.|

Reviewer: ```suggestion
| `force_flush_period`                | `500ms`                              | [Time](#time-parameters) since last time new data was found in the file, after which a partial log at the end of the file may be emitted.|
```
